configure.ac

```diff
@@ -158,7 +158,7 @@ AC_PROG_LIBTOOL
 AC_PROG_EGREP
 AC_PROG_SED
 # newer libtool...
-LT_PREREQ([2.4.3])
+LT_PREREQ([2.4.2])
```

Comment:
You should not have to change this

Reply:
I had to do so because I once got error with travis as if the libtool version was not corret. By changing the version to a lower number, I passed the travis compilation without the strange build failure.